src/components/ec-blog-item/ec-blog-item.vue

```diff
@@ -1,28 +1,34 @@
 <template>
-  <div class="ec-main-container ec-blog-item">
-    <div class="tw-grid tw-border-b tw-border-solid tw-border-gray-6">
+  <transition-group
+    class="ec-main-container ec-blog-item"
+    tag="div"
+  >
+    <div
+      v-for="post of posts"
+      :key="post.id"
+      class="tw-grid tw-border-b tw-border-solid tw-border-gray-6"
+      data-test="ec-blogItem__post"
```

Comment:
we use some special notation, take a look at some examples in this repo https://github.com/Ebury/chameleon/blob/master/src/components/ec-input-field/ec-input-field.vue